versions/3.1.0.md

```diff
@@ -147,12 +147,14 @@ Primitive data types in the OAS are based on the types supported by the [JSON Sc
 Note that `integer` as a type is also supported and is defined as a JSON number without a fraction or exponent part. 
 Models are defined using the [Schema Object](#schemaObject), which is a superset of JSON Schema Specification Draft 2019-09.
 
-<a name="dataTypeFormat"></a>Primitives have an optional modifier property: `format`.
-OAS uses several known formats to define in fine detail the data type being used.
+<a name="dataTypeFormat"></a>Primitives have an optional modifier property: `format`, which is defined by JSON Schema.
+OAS uses several known additional formats to define in fine detail the data type being used.
 However, to support documentation needs, the `format` property is an open `string`-valued property, and can have any value.
-Formats such as `"email"`, `"uuid"`, and so on, MAY be used even though undefined by this specification.
+Additional formats MAY be used even though undefined by either JSON Schema or this specification.
 Types that are not accompanied by a `format` property follow the type definition in the JSON Schema. Tools that do not recognize a specific `format` MAY default back to the `type` alone, as if the `format` is not specified.
 
+Note that by default, JSON Schema validators MUST NOT attempt to validate the `format` keyword.  It is primarily treated as an annotation that informs applications of the intended nature of the data.  For historical reasons, it is possible to configure JSON Schema validators to validate "format", but this is not consistently implemented and SHOULD NOT be relied upon in any environment where an unknown validator or unknown validator configuration might be used.
```

Comment:
Is there a reason for the double-quotes as opposed to backticks on the last mention of `"format"`?

Reply:
Nope, I'll fix that, thanks!